backend/server/schema.py

```diff
@@ -28,20 +36,130 @@ class Meta:
 
 
 class PredictionType(DjangoObjectType):
+    """Basic prediction type based on Prediction data model"""
+
     class Meta:
         model = Prediction
 
-    is_correct = graphene.Boolean()
 
+class ModelPredictionType(graphene.ObjectType):
+    """
+    Stats for predictions made by the given model through the given round
+    """
+
+    model_name = graphene.String()
+    cumulative_correct_count = graphene.Int()
+
+    def resolve_model_name(self, _info):
```

Comment:
I have seen that in some `def` you define the type of the return (for example in `def calculate_is_correct(cls, match: Match, predicted_winner: Team) -> bool:`) but here you don't ...if that for a reason?

Reply:
It's not strictly necessary, because 'mypy', like 'flow', can guess types most of the time, but for consistency, I should include return types.